fixes path of Crypt/CHAP.php.

--- php-pear-Auth-1.2.3.orig/Auth-1.2.3/Container/RADIUS.php	2004-12-04 21:39:21.815463240 +0100
+++ php-pear-Auth-1.2.3/Auth-1.2.3/Container/RADIUS.php	2004-12-04 21:40:02.488280032 +0100
@@ -110,7 +110,7 @@
                 $this->radius->chapid    = 1;
                 $this->radius->response  = pack('H*', $password);
             } else {
-                require_once 'Crypt_CHAP/CHAP.php';
+                require_once 'Crypt/CHAP.php';
                 $classname = 'Crypt_' . $this->authtype;
                 $crpt = new $classname;
                 $crpt->password = $password;
@@ -121,7 +121,7 @@
             }
 
         case 'MSCHAPv2':
-            require_once 'Crypt_CHAP/CHAP.php';
+            require_once 'Crypt/CHAP.php';
             $crpt = new Crypt_MSCHAPv2;
             $crpt->username = $username;
             $crpt->password = $password;
